api/src/main/java/edu/cornell/mannlib/vitro/webapp/dao/jena/IndividualDaoJena.java

```diff
@@ -52,7 +52,7 @@
 public class IndividualDaoJena extends JenaBaseDao implements IndividualDao {
 	//For random number generation, creating it everytime the method is called lead to nextInt being about the same
 	//if calls were made close together in time
-	private Random random = new Random(System.currentTimeMillis());
+    protected Random random = new Random(System.currentTimeMillis());
```

Comment:
```suggestion
    private Random random = new Random(System.currentTimeMillis());
```

Reply:
Thanks for that catch. I forgot to revert that when I was experimenting with some of the inheritance.